src/app/screens/PeerlyScreens/AppreciationScreen/appreciation.hooks.ts

```diff
@@ -1 +1,81 @@
-export function demohook() {}
+import {useMutation, useQuery} from 'react-query';
+import {
+  getCoreValuesList,
+  getCoworkerList,
+  postAppreciationRequest,
+} from '../../../services/PeerlyServices/appreciation';
+import {
+  PostAppreciationRequestBody,
+  GetCoworkersListRequest,
+  UserDetails,
+  CoreValue,
+} from '../../../services/PeerlyServices/appreciation/types';
+import toast from '../../../utils/toast';
+import {AxiosError} from 'axios';
+import {APIError} from './types';
+
+export function useGetCoworkerList(payload: GetCoworkersListRequest) {
+  const {data, isLoading, isError} = useQuery({
+    queryKey: ['coworker_list'],
```

Comment:
we might want to include contents of payload in queryKey to ensure we get new data when payload changes.
Please make a decision based on application logic.

Reply:
this is dropdown values and no dynamic changes will come so for that purpose we have not included the payload,